<launch>
      <param name="/use_sim_time" value="true"/>
    <node pkg="nodelet" type="nodelet" name="pickelhaube_kinect_camera_manager"
          args="manager"
          output="screen"
          required="true" >
       <param name="num_worker_threads" value="6" />
    </node>
  <!-- RESIZING -->
  <!-- <node pkg="nodelet"
        type="nodelet"
        args="load image_proc/crop_decimate /pickelhaube_kinect_camera_manager"
        name="kinect_resize" >
    <rosparam>
      decimation_x: 3
      decimation_y: 3
      height: 1280
      interpolation: 1
    </rosparam>
    <remap from="camera/image_raw" to="/rgb/image_raw"/>
    <remap from="camera/camera_info" to="/rgb/camera_info"/>
    <remap from="camera_out/image_raw" to="/rgb_resized/image_raw"/>
    <remap from="camera_out/camera_info" to="/rgb_resized/camera_info"/>
  </node> -->
  
  <node pkg="nodelet"
        type="nodelet"
        args="load image_proc/debayer /pickelhaube_kinect_camera_manager"
        name="kinect_grayscale" >
    <rosparam>
      decimation_x: 3
      decimation_y: 3
      height: 1280
      interpolation: 1
    </rosparam>
    <remap from="image_raw" to="/rgb/image_raw"/>
    <remap from="image_mono" to="/rgb/image_mono"/>
    <remap from="image_color" to="/rgb/image_color"/>
  </node>

  <node pkg="nodelet"
        type="nodelet"
        args="load image_proc/debayer /pickelhaube_kinect_camera_manager"
        name="cam2_grayscale" >
    <rosparam>
      decimation_x: 3
      decimation_y: 3
      height: 1280
      interpolation: 1
    </rosparam>
    <remap from="image_raw" to="/versavis/cam2/image_raw"/>
    <remap from="image_mono" to="/versavis/cam2/image_mono"/>
    <remap from="image_color" to="/versavis/cam2/image_color"/>
  </node>


  <!-- <node pkg="nodelet"
        type="nodelet"
        args="load image_proc/crop_decimate /pickelhaube_kinect_camera_manager"
        name="cam2_resize" >
    <rosparam>
      decimation_x: 2
      decimation_y: 2
      y_offset: 0
      height: 1200
      interpolation: 1
    </rosparam>
    <remap from="camera/image_raw" to="/versavis/cam2/image"/>
    <remap from="camera/camera_info" to="/pickelhaube/cam2/camera_info"/>
    <remap from="camera_out/image_raw" to="/pickelhaube_resized/cam2/image"/>
    <remap from="camera_out/camera_info" to="/pickelhaube_resized/cam2/camera_info"/>
  </node> -->

  </launch>
